src/quo2/components/dividers/strength_divider/view.cljs

```diff
@@ -0,0 +1,75 @@
+            [quo2.foundations.colors :as colors]
+            [react-native.linear-gradient :as linear-gradient]
+            [react-native.svg :as svg]
+            [utils.i18n :as i18n]
+            [quo2.components.icon :as icon]))
+
+(def status-list
+  {:very-weak   {:default-text (i18n/label :t/strength-divider-status-very-weak)
+                 :color        colors/danger-60
+                 :percentage   20}
+   :weak        {:default-text (i18n/label :t/strength-divider-status-weak)
+                 :color        (colors/custom-color :orange 60)
+                 :percentage   40}
+   :okay        {:default-text (i18n/label :t/strength-divider-status-okay)
+                 :color        (colors/custom-color :yellow 60)
+                 :percentage   60}
+   :strong      {:default-text (i18n/label :t/strength-divider-status-strong)
+                 :color        colors/success-60
+                 :percentage   80}
+   :very-strong {:default-text (i18n/label :t/strength-divider-status-very-strong)
+                 :color        colors/success-60
+                 :percentage   100}
+   :info        {:color colors/white-opa-40}
+   :alert       {:color colors/danger-60}})
+
+(def status-indicator-radius 6.5)
+
+(defn view
+  "Options
+   - `status` Status keyword (`:very-weak`/`:weak`/`:okay`/`:strong`/`:very-strong`/`:info`/`:alert`)
+
+   `text` message string(only works when status is `info`/`alert`)"
+  [{:keys [status] :or {status :very-weak}} text]
+  (let [{:keys [color percentage default-text]} (status-list status)
+        status-circumference                    (* 2 Math/PI status-indicator-radius)]
+    [linear-gradient/linear-gradient
+     {:colors [(colors/alpha color 0.05) (colors/alpha color 0)]}
+     [rn/view
+      {:style               style/container
+       :accessibility-label :strength-divider}
+      (when percentage
+        [svg/svg
+         {:view-box  "0 0 16 16"
+          :width     14.2
+          :transform [{:rotate "270deg"}]
+          :height    14.2}
+         [svg/circle
+          {:cx           8
+           :cy           8
+           :r            status-indicator-radius
+           :fill         :transparent
+           :stroke-width 1.2
+           :stroke       (colors/alpha color 0.2)}]
+         [svg/circle
+          {:cx                8
+           :cy                8
+           :r                 status-indicator-radius
+           :fill              :transparent
+           :stroke-width      1.2
+           :stroke-dasharray  status-circumference
+           :stroke-dashoffset (* (- 100 percentage) 0.01 status-circumference)
+           :stroke            color}]
+         [svg/circle]])
+      (when (= status :alert)
+        [icon/icon :i/alert
+         {:color color
+          :size  16}])
+      [text/text
+       {:size   :paragraph-2
+        :weight :medium
+        :style  (style/text color)} (if percentage default-text (or text default-text))]]]))
```

Comment:
This `if` could be simplified, because when there's not `percentage`, `default-text` doesn't exist in the `status-list` map. So I think it'd be the same if we put:

```clojure
(if percentage default-text text)
```
Also this `if` should be in the following line (76)